Week3/homework/3-step3.js

```diff
@@ -2,24 +2,37 @@
 
 // use a 'for' loop
 function repeatStringNumTimesWithFor(str, num) {
-  // add your code here
-  return str;
+  
+  let answerStr = "";
+  for(let i = 0; i < num; i++) {
```

Comment:
Aahaan. This works just fine. But have you thought about what happens if num < 0? Your initial declaration `let answerStr = '''` handles scenario where num could be equal to 0 but not if it less than it. Would you like to give it a try?

Reply:
if(num < 0) return "";  this would be the correct to do if the num is < 0... I have it already in the last loop and I forgot to add it in the first ones hahaha...